core/controllers/feedback_test.py

```diff
@@ -744,25 +731,10 @@ def test_get_num_threads_after_creating_feedback_analytics(self):
         feedback_services.create_thread(
             'exploration', self.exp_id, self.owner_id, 'subject', 'text')
 
-        feedback_analytics_aggregator_swap = self.swap(
-            feedback_jobs_continuous, 'FeedbackAnalyticsAggregator',
-            MockFeedbackAnalyticsAggregator)
-
-        with feedback_analytics_aggregator_swap:
-            (
-                feedback_jobs_continuous.FeedbackAnalyticsAggregator
-                .start_computation()
-            )
-            self.assertEqual(
-                self.count_jobs_in_mapreduce_taskqueue(
-                    taskqueue_services.QUEUE_NAME_CONTINUOUS_JOBS), 1)
-            self.process_and_flush_pending_mapreduce_tasks()
-            self.process_and_flush_pending_tasks()
-
-            response = self.get_json(
-                '%s/%s' % (feconf.FEEDBACK_STATS_URL_PREFIX, self.exp_id))
+        response = self.get_json(
+            '%s/%s' % (feconf.FEEDBACK_STATS_URL_PREFIX, self.exp_id))
 
-            self.assertEqual(response['num_total_threads'], 2)
-            self.assertEqual(response['num_open_threads'], 2)
+        self.assertEqual(response['num_total_threads'], 1)
```

Comment:
Actually I'm kind of confused. Why was it 2 before, and only 1 now?

Reply:
I think it was 2 before due to a bug, to be honest. Only one thread is being created by this function, so 1 is the only value that makes sense. From my understanding, the 2 comes from both of the realtime layers being assigned a value and subsequently combined.